addon/templates/components/polaris-text-field/connected.hbs

```diff
@@ -0,0 +1,21 @@
+{{#if (or left right)}}
+  <div class="Polaris-Connected">
+    {{#if left}}
+      <div class="Polaris-Connected__Item Polaris-Connected__Item--connection">
+        {{render-content left}}
+      </div>
+    {{/if}}
+
+    <div class="Polaris-Connected__Item Polaris-Connected__Item--primary">
+      {{yield}}
+    </div>
+
+    {{#if right}}
+      <div class="Polaris-Connected__Item Polaris-Connected__Item--connection">
+        {{render-content right}}
+      </div>
+    {{/if}}
+  </div>
+{{else}}
+  {{yield}}
+{{/if}}
```

Comment:
Can you move this out into a separate `polaris-connected` component please, as per the React library? If you feel like it you could even do it under a separate PR and rebase this branch onto it 😛